include/swift/AST/DiagnosticsSema.def

```diff
@@ -9060,5 +9060,8 @@ ERROR(unsafe_self_dependent_result_attr_on_invalid_decl,none,
 
 REMARK(macro_expansion_line, none, "macro content: |%0|", (StringRef))
 
+WARNING(empty_catch_block, none, "empty catch block silently ignores errors; consider using 'try?' or 'catch _' to explicitly ignore", ())
```

Comment:
tiny nitpick: I think I'd rather we not encourage developers to reach for `try?` when they've already gone through the hassle of wrapping in a do/catch, personally. Other folks in the community may feel differently.